java/compat/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Symlinks under Git on Windows are currently an experimental feature, so we can't rely on them working. I'm ok with the redundancy for now.